test-cypress/cypress.json

```diff
@@ -8,6 +8,6 @@
     "supportFile": "support/index.js",
     "videosFolder": "report/videos",
     "video": false,
-    "defaultCommandTimeout": 30000,
+    "defaultCommandTimeout": 10000,
```

Comment:
Not sure why this was set to 30seconds but I think it's far too big timeout for the commands. Will leave this way for now and see if it'll cause some issues with the builds.